src/main/views/courts/tabs/casesHeardContent.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<h2 class="govuk-heading-l">Cases Heard</h2>
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if updated === true %}
+  {{ govukPanel({
+    titleText: "Cases heard updated"
+  }) }}
+{% endif %}
+
+{% if allAreasOfLaw|length %}
+
+  {% set courtAreasOfLawNames = [] %}
+  {% for areaOfLaw in courtAreasOfLaw %}
+    {% set allAreasOfLawItems =
+      (courtAreasOfLawNames.push(areaOfLaw.name), courtAreasOfLawNames) %}
+  {% endfor %}
+
+  {% set allAreasOfLawItems = [] %}
+  {% for areaOfLaw in allAreasOfLaw %}
+     {% if areaOfLaw.name in courtAreasOfLawNames %}
+        {% set allAreasOfLawItems =
+          (allAreasOfLawItems.push({
+            attributes: { 'data-inputType': 'cases-heard' },
+            'text': areaOfLaw.name,
+            'value': areaOfLaw.name,
+            'id': areaOfLaw.id,
+            checked: true
+          }), allAreasOfLawItems) %}
+      {% else %}
+        {% set allAreasOfLawItems =
+          (allAreasOfLawItems.push({
+            attributes: { 'data-inputType': 'cases-heard' },
+            'text': areaOfLaw.name,
+            'value': areaOfLaw.name,
+            'id': areaOfLaw.id,
+            checked: false
```

Comment:
I think only the checked value is different in the 'if' and 'else'. Is it possible to remove the if/else and just use one setter by setting a boolean property based on whether the areaOfLaw.name is in the courtAreaOfLawNames and then using this to set the 'checked' value?